Assets/Powerup Jump.prefab

```diff
@@ -0,0 +1,269 @@
+%YAML 1.1
```

Comment:
Please put your prefabs in the correct subfolders. Don't dump stuff into the root assets directory

Reply:
Actually it shouldn't even be there LOL lemme just remove it